mlflow/gluon.py

```diff
@@ -306,7 +311,8 @@ def log_model(
 
 
 @experimental
-def autolog(log_models=True):
+@autologging_integration(FLAVOR_NAME)
+def autolog(log_models=True, disable=False):  # pylint: disable=unused-argument
     """
     Enable automatic logging from Gluon to MLflow.
```

Comment:
Nit: Can we update the docstring here with the enables / disables / configures text?

Reply:
Fixed: https://github.com/mlflow/mlflow/pull/3854/commits/530dd20aa96ca484ca185ee347d003562ccfcfda!